Common/UnitEnumValues.g.json

```diff
@@ -965,7 +965,10 @@
     "TonneSquareCentimeter": 25,
     "TonneSquareDecimeter": 26,
     "TonneSquareMeter": 27,
-    "TonneSquareMilimeter": 28
+    "TonneSquareMilimeter": 28,
+    "KilotonneSquareMillimeter": 30,
+    "MegatonneSquareMillimeter": 37,
+    "TonneSquareMillimeter": 32
```

Comment:
This maybe a little hack-y, but I think that if we changed these by hand (to the old values) - I think they would _stick_.

Reply:
For fixing typos, we can definitely tweak these by hand to match the old values 👍

Reply:
fixed in https://github.com/angularsen/UnitsNet/pull/1490/commits/23a68bc58f0ce5c6aff630ec721368b4730f6ca7